src/components/SignUp/SignUpScreen.js

```diff
@@ -70,99 +287,416 @@ export default function SignUpScreen() {
+    return msg;
+  },
+  [formState]);
+
+  function Step1() {
+    if (currentStep !== 1) {
+      return null;
+    }
+    return [
+      <div className={classes.subtitleText}>
+        Are you a...
+      </div>,
+      <Grid
+        container
+        spacing={2}
+        direction="row"
+        justify="center"
+        alignItems="stretch"
+      >
+        {BUTTONS.map((bt) => (
+          <Grid
+            item
+            className={classes.card}
+            component={Card}
+            xs={12}
+            sm={6}
+            md={3}
+            key={BUTTONS.indexOf(bt)}
+          >
+            <CardActionArea onClick={roleChange} value={bt.id}>
+              <CardMedia
+                image={bt.image}
+                style={bt.fruit}
+              />
+              <CardContent>
+                <Typography className={classes.labelText}>
+                  {bt.title}
+                </Typography>
+              </CardContent>
+            </CardActionArea>
           </Grid>
-          <Grid item xs={12}>
-            <InputLabel
-              style={{ disableAnimation: false }}
-              disableAnimation={false}
-              htmlFor="searchCriteria"
-            >
-              Role
-            </InputLabel>
-            <Select
-              className="text-fields"
-              labelId="demo-simple-select-label"
-              id="demo-simple-select"
-              value={role}
-              onChange={roleChange}
-              disabled={loading}
+        ))}
+      </Grid>,
+    ];
+  }
+
+  function Step4() {
+    if (currentStep !== 4) {
+      return null;
+    }
+    return [
+      <div>
+        <div className={classes.cenSubtitleText}>
+          Account Created!
+        </div>
+        <div className={classes.cenRegSubtitleText}>
+          { role === 1 ? 'Please await authentication from Farm2People.'
+            : 'Farm2People will be in touch with you shortly.'}
+        </div>
+      </div>,
+      <Grid
+        container
+        spacing={2}
+        alignItems="center"
+      >
+        <Grid item xs={12}>
+          <Box mt={5}>
+            <Button
+              type="button"
+              className={classes.submitButton}
+              color="primary"
+              variant="contained"
+              style={{ backgroundColor: '#53AA48' }}
+              onClick={onPrev}
             >
-              <MenuItem value="vendor">Vendor</MenuItem>
-              <MenuItem value="buyer">Buyer</MenuItem>
-              <MenuItem value="agency">Agency</MenuItem>
-            </Select>
-          </Grid>
-          {errorMsg && <Grid item xs={12} className="error-msg">{errorMsg}</Grid>}
-          {repeatPwdCheck && <Grid item xs={12} className="error-msg">{repeatPwdCheck}</Grid>}
-          <Grid item xs={12}>
-            <Button disabled={isInvalid || loading} className="text-fields" variant="contained" color="primary" type="submit">
-              Sign Up
+              Done
```

Comment:
When you press Done, don't forget to do history.push() to take you to the landing page!